packages/theme/src/hooks/useTabs.tsx

```diff
@@ -29,7 +30,7 @@ export function useTabs<T extends string>(defaultTab: T, ...possibleTabs: T[]) {
     }, [isCurrentTabAvailable, defaultTab])
 
     const onChange = useCallback((event: unknown, value: any) => {
-        setTab(value)
+        startTransition(() => setTab(value))
```

Comment:
actually, you need to see the tab switch immediately, but the tab content to be loaded with the transition. if you want to use useTransition, you should split it into two state, or use `useDeferredValue`.